int32 tiempoEntrada
float32 velEntrada
---
bool continua
int32 tiempoMovimiento